src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
Not ScrollableListView, but on `MenuControl`>`panelsScrollPane`. It's a JavaFX ScrollPane object. If I want to see its pane position, I need to create a public method in `MenuControl` to see the scroll position, but that method's only purpose is for my test and nothing else, which I think is not a good design.

I will see if I can do the first issue selected test. The purpose would be different from the TODO I wrote, though. The TODO is because the focused panel was outside of view (say the focused panel is the first panel but the view is stuck on last panel, hence user is doing things without looking at the panel he is working on). I thought a test on this would be good, but I need to access the ScrollPane value.